File: abc/abc163/abc163_c.fs
// https://atcoder.jp/contests/abc163/submissions/12146120

let N = stdin.ReadLine() |> int

let A =
    stdin.ReadLine().Split()
    |> Array.map int
    |> Array.countBy id
    |> dict

[| 1..N |]
|> Array.map (fun x ->
    if A.ContainsKey x then A.[x]
    else 0)
|> Array.iter stdout.WriteLine

File: abc/abc163/abc163_c.cs
// https://atcoder.jp/contests/abc163/submissions/12145128

using System;
using System.Collections.Generic;
using System.Linq;

class Program
{
    static void Main()
    {
        var N = int.Parse(Console.ReadLine());
        var A = Console.ReadLine().Split().Select(int.Parse).ToArray();

        var b = new Dictionary<int, int>();
        for (var i = 0; i < A.Length; i++)
        {
            if (b.ContainsKey(A[i]))
            {
                b[A[i]]++;
            }
            else
            {
                b.Add(A[i], 1);
            }
        }

        for (var i = 1; i < N + 1; i++)
        {
            if (b.ContainsKey(i))
            {
                Console.WriteLine(b[i]);
            }
            else
            {
                Console.WriteLine(0);
            }
        }
    }
}
